dahdi-tools.spec

```diff
@@ -114,3 +131,21 @@ cd $RPM_BUILD_DIR
 %defattr(-, root, root)
 %{_includedir}/dahdi/tonezone.h
 %{_libdir}/libtonezone.a
+
+%files libs
+%defattr(-,root,root,-)
+%doc LICENSE LICENSE.LGPL
+%{_libdir}/*.so.*
+
+%files devel
+%defattr(-,root,root,-)
+%doc LICENSE LICENSE.LGPL
+%{_includedir}/*
+%{_libdir}/*.so
```

Comment:
Are you sure these files shouldn't be included inside the `libs` package?

Reply:
you're right